### PR TITLE
Enh/Enhance group information display

### PR DESCRIPTION
**Pull request by GVodyanov**
Fix https://github.com/nextcloud/contacts/issues/3706

| Before | After |
| - | - |
| image | image |

### CODE REVIEW

File: src/components/Properties/PropertyGroups.vue [hunk 1]
```diff
@@ -55,9 +54,15 @@
 						{{ t('contacts', 'No results') }}
 					</span>
 				</NcSelect>
-				<p v-else>
-					{{ localValue.length === 0 ? t('contacts','None'): localValue.toString() }}
-				</p>
+				<div v-else>
+					<span v-if="localValue.length === 0">{{ t('contacts','None') }}</span>
+
+					<div class="group__list">
```
Comment (hamza221) [inline src/components/Properties/PropertyGroups.vue:60]:
```suggestion
					<div v-else class="group__list">
```
so we don't get an empty div in the dom

**Review by hamza221 — CHANGES_REQUESTED** [review]

**Comment by hamza221** [comment]
@GVodyanov can you please check failing CI ^^

**Comment by hamza221** [comment]
@GVodyanov friendly ping

**Comment by GVodyanov** [comment]
Sorry here I am, btw during testing I found this https://github.com/nextcloud/contacts/issues/3880, looks pretty bad @hamza221

**Review by hamza221 — APPROVED** [review]

**Comment by hamza221** [comment]
@ChristophWurst  Is this the desired UI?

**Comment by ChristophWurst** [comment]
Seems find if I look at the screenshots of https://github.com/nextcloud/contacts/issues/3706

**Pull request merged**